### PR TITLE
add javascript piechart

### PR DESCRIPTION
**Pull request by emrekuecuek**
for #8 , add piechart

### CODE REVIEW

File: templates/index.php [hunk 1]
```diff
@@ -1,22 +1,13 @@
 <?php
```
Comment (karakayasemi) [inline templates/index.php]: HTML olarak baslayip gerekli yerlerde php ac kapa bence

Comment (karakayasemi) [reply templates/index.php]: echo "div" yapmamis olursun, kotu gozukuyor

File: js/main.js [hunk 2]
```diff
@@ -13,13 +19,17 @@ $(document).ready(function () {
                     yValueFormatString: "##0.00\"%\"",
                     indexLabel: "{label} {y}",
                     dataPoints: [
-                        {y: 100, label: ""},
+                        {y: 10, label: "asd"}
```
Comment (karakayasemi) [inline js/main.js]: asd?

**Review by karakayasemi — CHANGES_REQUESTED** [review]
See my comments. Change it and ping me again

File: js/main.js (hunk 2 above)
Comment (emrekuecuek) [reply js/main.js]: Deneme yapıyordum javascript'te, gözümden kaymış

File: lib/Controller/PageController.php [hunk 3]
```diff
@@ -35,6 +35,7 @@ public function __construct(
     }
 
     /**
+     * @NoCSRFRequired
```
Comment (emrekuecuek) [inline lib/Controller/PageController.php:38]: NoCSRFRequired also needed here, too.

**Comment by karakayasemi** [comment]
Squash your commits to one commit. Code looks okay. I will try it tomorrow.

**Pull request closed without merging**